src/lib/storage/value_segment.hpp

```diff
@@ -13,6 +13,8 @@ namespace opossum {
 template <typename T>
 class ValueSegment : public BaseSegment {
  public:
+  ValueSegment() = default;
```

Comment:
I think we should undo this, we're probably only expected to change code wherever
```
// Implementation goes here
```
was stated

Reply:
Yes. 
As long as we don't have a constructor, this line does also not change anything, right?

Reply:
Yes, it will still be implicitly defined.